src/pages/ProductPage/ProductPage.jsx

```diff
@@ -8,10 +8,14 @@ export function ProductPage() {
   const [products, setProducts] = useState([]);
 
   async function getProducts() {
-    setLoader(true);
-    const { data } = await axios.get("./api/products");
-    setLoader(false);
-    setProducts(data.products);
+    try {
+      setLoader(true);
+      const { data } = await axios.get("./api/products");
+      setLoader(false);
+      setProducts(data.products);
+    } catch (error) {
```

Comment:
Later on you can display here error as well, for now you can just console.log the error.